Maestro/subscriptions.json

```diff
@@ -1200,20 +1203,22 @@
     //     }
     //   }
     // },
+    // Reviewers: Could someone remind me why we don't do automatic merges through 'blazor-wasm' in aspnetcore-tooling and blazor?
     // Automate opening PRs to merge aspnet release/3.1 branches back to master
     {
       "triggerPaths": [
-        "https://github.com/aspnet/AspNetCore-Tooling/blob/release/3.1//**/*",
-        "https://github.com/aspnet/Blazor/blob/release/3.1//**/*",
-        "https://github.com/aspnet/EntityFramework6/blob/release/6.4//**/*",
-        "https://github.com/aspnet/EntityFrameworkCore/blob/release/3.1//**/*",
-        "https://github.com/aspnet/Extensions/blob/release/3.1//**/*"
+        "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.1//**/*",
+        "https://github.com/dotnet/blazor/blob/release/3.1//**/*",
+        "https://github.com/dotnet/ef6/blob/release/6.4//**/*",
+        "https://github.com/dotnet/efcore/blob/release/3.1//**/*",
+        "https://github.com/dotnet/extensions/blob/release/3.1//**/*"
       ],
       "action": "github-dnceng-branch-merge-pr-generator",
       "actionArguments": {
         "vsoSourceBranch": "master",
         "vsoBuildParameters": {
-          "GithubRepoOwner": "aspnet",
+          // Reviewers: Is changing this setting problematic here -- where the source repos are all in the same org?
+          "GithubRepoOwner": "dotnet",
```

Comment:
Yes, this is used by the script that does the merging.  These rules will need to be split

Reply:
I'm confused. Why not use `dotnet` here? The source repos for this rule are now all in that org.

Reply:
@mmitche❔

Reply:
Yep you're right.